docs/modules.md

```diff
@@ -0,0 +1,52 @@
+# Modules
+
+This guide is intended for developers and provides an overview of the major
+modules within the Simperby project.
+
+The project consists of 4 major modules:
+
+1. Repository: This module manages the local repository of a node, which tracks
+   the canonical chain of the network.
+2. Governance: This module manages a node's client in the p2p governance voting
+   protocol.
+3. Consensus: This module manages a node's state machine of the BFT algorithm
+   (Vetomint).
```

Comment:
How about
```suggestion
   which is Vetomint.
```
?? (just a personal suggestion)

Reply:
I think what Junha wrote is better.